modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/SkinCleanupTest.java

```diff
@@ -274,6 +274,8 @@ public void testTreeTableRowFixedCellSizeEnabled() {
         assertTrue("fixed cell size enabled", isFixedCellSizeEnabled(tableRow));
     }
 
+    @Ignore("JDK-8295809") // TODO probably need to verify the result of listener action,
```

Comment:
We need to file a follow-on bug to modify or cleanup these ignored tests in future.
Use the bug ID of the newly filed bug with `@Ignore` tag for these two tests.